src/main/java/dev/andrylat/vomelianchuk/card_number_operations/CardValidationException.java

```diff
@@ -0,0 +1,24 @@
+package dev.andrylat.vomelianchuk.card_number_operations;
+
+import java.util.List;
+
+public class CardValidationException extends RuntimeException {
+    private final List<String> errorsMessage;
+
+    public CardValidationException(List<String> errorsMessage) {
+        this.errorsMessage = errorsMessage;
+    }
+
+    public List<String> getErrorsMessage() {
+        return errorsMessage;
+    }
+
+    public String getErrorsMessagesInLine() {
```

Comment:
Exception should not have responsibility for formatting messages. It is responsibility of class who uses it.

Reply:
Fixed here: 7a87509